sdk/src/AbstractMain.cpp

```diff
@@ -230,14 +232,17 @@ int AbstractMain::run(int in_argc, char** in_argv)
 
    if (options.getLogLevel() > LogLevel::INFO)
    {
-      addLogDestination(
+       
+        addLogDestination(
          std::unique_ptr<ILogDestination>(
             new FileLogDestination(
-               3,
+               "3",
                options.getLogLevel(),
+               LogMessageFormatType::PRETTY,
                getProgramId(),
-               options.getScratchPath())));
-   }
+               logging::FileLogOptions(options.getScratchPath())
```

Comment:
```suggestion
               logging::FileLogOptions(options.getLoggingDir())
```

Let's use the new logging directory parameter here instead of the scratch path.